jenkins/internal-ci/generic/branch/cortx-cc.groovy

```diff
@@ -0,0 +1,208 @@
+                    CORTX_UTILS_REPO_OWNER=$(echo $CORTX_UTILS_URL | cut -d "/" -f4)
+                    yum erase python36-PyYAML -y
+                    yum install -y gcc python3 python3-pip python3-devel python3-setuptools openssl-devel libffi-devel python3-dbus
+                    pip3 install  --no-cache-dir --trusted-host cortx-storage.colo.seagate.com -i http://cortx-storage.colo.seagate.com/releases/cortx/third-party-deps/python-deps/python-packages-$version-latest/ -r https://raw.githubusercontent.com/Seagate/cortx-utils/${branch}/py-utils/python_requirements.txt -r https://raw.githubusercontent.com/Seagate/cortx-utils/${branch}/py-utils/python_requirements.ext.txt
+                    rm -rf /etc/pip.conf
+                '''
+
+                sh label: 'Install Dependencies', script: '''
+                    set +x
+                    yum-config-manager --add-repo=http://cortx-storage.colo.seagate.com/releases/cortx/github/main/$os_version/$release_tag/cortx_iso/
+                    yum-config-manager --save --setopt=cortx-storage*.gpgcheck=1 cortx-storage* && yum-config-manager --save --setopt=cortx-storage*.gpgcheck=0 cortx-storage*
+                    yum clean all;rm -rf /var/cache/yum
+                    yum install cortx-py-utils cortx-motr{,-devel} -y --nogpgcheck
+                '''
+            }
+        }
+
+        stage('Build') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                sh label: 'Build', returnStatus: true, script: '''
+                    set -xe
+                    pushd $component
+                    echo "Executing build script"
+                    export build_number=${BUILD_ID}
+                    make rpm
+                    popd
+                '''    
+            }
+        }
+
+        stage ('Upload') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                sh label: 'Copy RPMS', script: '''
+                    mkdir -p $build_upload_dir/$BUILD_NUMBER
+                    cp /root/rpmbuild/RPMS/x86_64/*.rpm $build_upload_dir/$BUILD_NUMBER
+                '''
+                sh label: 'Repo Creation', script: '''pushd $build_upload_dir/$BUILD_NUMBER
+                    rpm -qi createrepo || yum install -y createrepo
+                    createrepo .
+                    popd
+                '''
+
+            }
+        }
+            
+        stage ('Tag last_successful') {
+            when { not { triggeredBy 'UpstreamCause' } }
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                sh label: 'Tag last_successful', script: '''pushd $build_upload_dir/
+                    test -L $build_upload_dir/last_successful && rm -f last_successful
+                    ln -s $build_upload_dir/$BUILD_NUMBER last_successful
+                    popd
+                '''
+            }
+        }
+
+        // stage ("Release") {
```

Comment:
We will enable this after confirmation from CC team right.

Reply:
Yes